k8s/bigchaindb/bigchaindb-dep.yaml

```diff
@@ -4,46 +4,28 @@
 # separate pod                                                #
 ###############################################################
 
-apiVersion: v1
-kind: Service
-metadata:
-  name: bdb-svc
-  namespace: default
-  labels:
-    name: bdb-svc
-spec:
-  selector:
-    app: bdb-dep
-  ports:
-  - port: 9984
-    targetPort: 9984
-    name: bdb-port
-  type: ClusterIP
-  clusterIP: None
----
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: bdb-dep
+  name: bdb-instance-0-dep
 spec:
   replicas: 1
   template:
     metadata:
       labels:
-        app: bdb-dep
+        app: bdb-instance-0-dep
     spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: bigchaindb
-        image: bigchaindb/bigchaindb:master
+        image: bigchaindb/bigchaindb:0.10.1
```

Comment:
The comment at the top of this file says "This config file runs bigchaindb:master as a k8s Deployment...". Maybe just change that comment to "This config file runs BigchainDB Server as a k8s Deployment..." (i.e. with no explicit mention of a version number).